### DISCUSSION TITLE
Promote an inverse relation?

### DISCUSSION THREAD

**Discussion by manyworldspublishing** [TriliumNext/Trilium#3609] [General]
Hi folks,

I've promoted a relation and would like to promote its inverse as well. When I go to the target note and look at owned attributes, I see the inverse relation but promoting is not an option.

Thanks,
-david

**Comment by zadam** [comment]
Hi, if you want to have the attribute promoted also on the target, the promoted definition needs to be on the target note as well.

**Comment by manyworldspublishing** [comment]
Thank you!
________________________________

**Comment by manyworldspublishing** [comment]
@zadam , I defined the relation #relation:HasResource="multi,inverse=ResourceOf" on a single note and the proper backlinks were generated. The relation "HasResource" is then offered as a tooltip on other notes, but the backlinks are not generated for those other notes. Is there a way to define the relation once and have it behave in an identical way for all notes that employ it?

**Reply by zadam** [reply]
`#relation:HasResource="multi,inverse=ResourceOf"` has to be defined on the note as well for the relation definition to take effect.

Since doing that manually would be tiresome, some form of [attribute inheritance](https://github.com/zadam/trilium/wiki/Attribute_Inheritance) is used - most typically template.

**Comment by manyworldspublishing** [comment]
Thanks. I got a template working!
________________________________
